integration-tests/src/test/scala/play/libs/ws/ahc/AhcWSClientSpec.scala

```diff
@@ -99,10 +99,11 @@ class AhcWSClientSpec(implicit executionEnv: ExecutionEnv) extends Specification
         val responseXml = response.getBody(xml())
         responseXml.normalizeDocument()
 
-        responseXml.isEqualNode(document) must beTrue
+        responseXml.isEqualNode(document) must beTrue and {
```

Comment:
There is quite no value to test getters depending on the mock used in the unit tests